tests/libvmifact/factory.go

```diff
@@ -146,3 +148,23 @@ func NewWindows(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 	vmi.Spec.Domain.Firmware = &kvirtv1.Firmware{UUID: WindowsFirmware}
 	return vmi
 }
+
+func NewTinyOSVMWithDataVolume(dv *cdiv1.DataVolume, extraVMOpts ...libvmi.VMOption) *kvirtv1.VirtualMachine {
+	// Supplied with no user data, AlpimeWithTestTooling image takes more than 200s to allow login
+	withNonEmptyUserData := libvmi.WithCloudInitNoCloudEncodedUserData("#!/bin/bash\necho hello\n")
```

Comment:
super nit: I would define this directly in the VM definition without the need of the extra-variable

Reply:
nit indeed: I actually prefer this added variable. In first glance, I had no idea why we pass this `echo hello`.